### PR TITLE
Added ERC721_Pausable

### LINKED ISSUE #114: Add ERC721 Pausable preset

### PR DESCRIPTION
**Pull request by andrew-fleming**
# Added ERC721_Pausable

## Summary

This PR resolves #114 and includes the pausable preset for ERC721. Further, `develop` is the origin branch; therefore, #108 should be merged first.

### CODE REVIEW

File: tests/test_ERC721_Pausable.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            other.contract_address,
+            *third_token_id
+        ])
+    )
+
+
+@pytest.mark.asyncio
+async def test_unpause(erc721_factory):
+    _, erc721, owner, other, erc721_holder = erc721_factory
+
```
Comment (martriay) [inline tests/test_ERC721_Pausable.py:125]: maybe testing that it's paused? just to ensure a proper state before testing

Comment (martriay) [reply tests/test_ERC721_Pausable.py:125]: we really should refactor our test suites to make each case independant

**Review by martriay — APPROVED** [review]
Left a few non-blocking suggestions

File: tests/test_ERC721_Pausable.py (hunk 1 above)
Comment (andrew-fleming) [reply tests/test_ERC721_Pausable.py:115]: Excellent idea! I made the test cases independent here by changing the 721 factory scope to `function`. The one downside, however, consists of how long the tests take. I was also tempted to add `conftest.py` to keep account and contract fixtures, but I think it'd be more appropriate when we focus on reorganizing the repo

Comment (martriay) [reply tests/test_ERC721_Pausable.py:115]: we can have a memoized function to reduce cost

**Pull request merged**